styles/utils.css

```diff
@@ -17,5 +17,87 @@
     --primary-button-hover: rgb(21, 50, 180);
     --secondary-button-hover: rgb(236, 233, 233);
     --secondary-hover: rgb(220, 219, 245);
-    --text-highlight: #ddd7d7;
+
+
+}
+
+/* width and height */
+.w-10{
+  width: 10rem;      
+}
+
+.w-20{
+    width: 20rem;
+}
+
+.w-30{
+    width: 30rem;
+}
+
+.w-40{
+    width: 40rem;
+}
+
+.w-50{
+    width: 50rem;
+}
+
+.w-60{
+    width: 60rem;
+}
+
+.w-70{
+    width: 70rem;
+}
+
+.w-80{
+    width: 80rem;
+}
+
+.w-90{
+    width: 90rem;
+}
+
+.w-100{
+    width: 100rem;
+}
+
+.h-10{
+    width: 10rem;
+}
+
+.h-20{
+    width: 20rem;
+}
+
+.h-30{
+    width: 30rem;
+}
+
+.h-40{
+    width: 40rem;
+}
+
+.h-50{
+    width: 50rem;
+}
+
+.h-60{
+    width: 60rem;
+}
+
+.h-70{
+    width: 70rem;
+}
+
+.h-80{
+    width: 80rem;
+}
+
+.h-90{
+    width: 90rem;
+}
+
+.h-100{
+    width: 100rem;
 }
```

Comment:
you have given width property to height classes

Reply:
Oh no, I think I mixed few things while resolving the merge conflicts. Will fix this!